docs/pipelines/process/variables.md

```diff
@@ -364,6 +364,36 @@ jobs:
     name: echovar
 ```
 
+The output variables of a [deployment](deployment-jobs.md) job also need to be prefixed with the job name (in this case, `A`):
+
+```yaml
+jobs:
+
+# Set an output variable from a deployment
+- deployment: A
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
```

Comment:
Thanks @Victor-Savu! I found that I had to define an environment here for my YAML to successfully run. The environment should have with the same indent level as strategy and be directly above strategy. Can you add it?

![Screen Shot 2019-11-08 at 10 38 20 AM](https://user-images.githubusercontent.com/64906/68488844-ee11ce80-0213-11ea-969d-a6efb590c948.png)

Reply:
Thank you for the review, @juliakm! I added the environment field. Is it OK that I named it `staging`?

Reply:
Thanks @Victor-Savu. That's great!